src/main/java/seedu/address/ui/FeedPostCard.java

```diff
@@ -31,22 +31,18 @@
     @FXML
     private Label source;
     @FXML
-    private Label title;
-    @FXML
-    private Hyperlink address;
+    private Hyperlink link;
 
     public FeedPostCard(FeedPost feedPost) {
         super(FXML);
         this.feedPost = feedPost;
 
         source.setText(feedPost.getSource());
-        title.setText(feedPost.getTitle());
-        address.setText(feedPost.getAddress());
```

Comment:
nice, most users dont care about the address, so its good to combine the 2 together